R/auto.R

```diff
@@ -206,7 +206,7 @@ update_cran_comments <- function() {
 }
 
 get_crp_date <- function() {
-  cmt <- gh::gh("/repos/eddelbuettel/crp/commits")[[1]]
+  cmt <- gh("/repos/eddelbuettel/crp/commits")[[1]]
```

Comment:
this change is for being able to mock it while respecting the guidance in https://testthat.r-lib.org/reference/local_mocked_bindings.html#namespaced-calls